src/Controller/BaseController.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+
+class BaseController extends AbstractController
+{
+    protected function isReadOnly(): bool
+    {
+        return strpos($this->container->get('request_stack')->getCurrentRequest()->get('_route'), 'read_') === 0 ? true : false;
```

Comment:
The `===` will result in true or false, so you can remove ` ? true : false`.

And clarity, could you assign `$this->container->get('request_stack')->getCurrentRequest()->get('_route')` to a variable, first?

So, something like:
```php
        $route = $this->container->get('request_stack')->getCurrentRequest()->get('_route');
        return strpos($route, 'read_') === 0;
```